common/pom.xml

```diff
@@ -28,6 +28,11 @@
     <artifactId>common</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Is the log4j dependency change related to the current issue?

Reply:
adjust project configuration so that each module can be compiled and tested independently, and the scope is provided